### PR TITLE
docs: update metric tables and hyperparameters for clip finetuning

### PR DESCRIPTION
**Pull request by guenthermi**
The evaluation metric scores for finetuning clip where not correct. Now, I used a bigger portion of the fashion captioning dataset for finetuning, slightly changed the hyperparmeter to values which work better and update the evaluation metric scores

---

- [ ] This PR references an open issue
- [x] I have added a line about this change to CHANGELOG

### CODE REVIEW

**Review by bwanglzu — APPROVED** [review]
LGTM! It's pain to see we're not that effective  :)
